akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -576,8 +613,14 @@ final class ByteStringBuilder extends Builder[Byte, ByteString] {
         _length += bs.length
       case xs: WrappedArray.ofByte ⇒
         putByteArrayUnsafe(xs.array.clone)
+      case seq: collection.IndexedSeq[Byte] if shouldResizeTempFor(seq.length) ⇒
+        //we will have to resize and copy anyways, might as well flush it
+        clearTemp()
+        _builder += ByteString.ByteString1(seq)
+        _length += seq.length
```

Comment:
perhaps due to food coma, but I currently don’t see why this case should be added